requirements.txt

```diff
@@ -1,5 +1,3 @@
-beautifulsoup4
-requests
-cyclonedx-python-lib
-pytest
-black
\ No newline at end of file
+beautifulsoup4==4.12.3
+cyclonedx-python-lib===7.3.1
+Requests==2.31.0
```

Comment:
should this be capitalized? https://pypi.org/project/requests/ here the install command is lower case

Reply:
I'll change it, weirdly `pipreqs` generated it like that

Reply:
maybe both work